src/convert/replacements.ts

```diff
@@ -195,7 +195,8 @@ export const getReplacements = async (
 export const matchesFile = (f: string, r: ReplacementConfig): boolean =>
   // filenames will be absolute.  We don't have convenient access to the pkgDirs,
   // so we need to be more open than an exact match
-  Boolean((r.filename && posixifyPaths(f).endsWith(r.filename)) || (r.glob && minimatch(f, `**/${r.glob}`)));
+  (typeof r.filename === 'string' && posixifyPaths(f).endsWith(r.filename)) ||
```

Comment:
undefined && something => undefined

So it's possible this could return undefined if they're both undefined.